src/Neo/Cryptography/Base58.cs

```diff
@@ -99,20 +118,35 @@ public static string Encode(ReadOnlySpan<byte> input)
             BigInteger value = new(input, isUnsigned: true, isBigEndian: true);
 
             // Encode BigInteger to Base58 string
-            var sb = new StringBuilder();
+            var sb = new StringBuilder(input.Length * 138 / 100 + 5);
```

Comment:
Could you add a comment about how did you get this value?

Reply:
It's roughly estimated end length value for base58 characters.